akka-bbb-apps/src/main/scala/org/bigbluebutton/core/apps/whiteboard/ClearWhiteboardPubMsgHdlr.scala

```diff
@@ -1,15 +1,16 @@
 package org.bigbluebutton.core.apps.whiteboard
 
-import org.bigbluebutton.core.running.LiveMeeting
 import org.bigbluebutton.common2.msgs._
-import org.bigbluebutton.core.bus.MessageBus
 import org.bigbluebutton.core.apps.{ PermissionCheck, RightsManagementTrait }
+import org.bigbluebutton.core.bus.MessageBus
+import org.bigbluebutton.core.running.LiveMeeting
 
 trait ClearWhiteboardPubMsgHdlr extends RightsManagementTrait {
   this: WhiteboardApp2x =>
 
   def handle(msg: ClearWhiteboardPubMsg, liveMeeting: LiveMeeting, bus: MessageBus): Unit = {
 
+    /*

```

Comment:
Brief check shows that `ClearWhiteboardPubMsg` is used in the recording area. Are you sure you want to comment this out rather than to call it below?
I don't quite understand the context though.
>log.error("Ignoring message ClearWhiteboardPubMsg since this functions is not available in the new Whiteboard")

cc @KDSBrowne do you know?

Reply:
I commented (instead of remove) because I thought it could be necessary in the future 😅

I didn't know the reason this message was not being sent anymore. Good to know it's expected by recordings.